File: tools/app-map-mcp/fixtures/lint/Bad.kt
// fixtures/lint/Bad.kt — lint-ids fixture (01 R8). Every violation is deliberate.
package com.example.app.invoices

import androidx.compose.foundation.layout.Box
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.platform.testTag
import com.example.appmap.AppMapId

@Composable
fun InvoiceListScreen() {
  Box(Modifier.testTag("screen.invoice_list")) {                   // string_literal_id (must be AppMapId.Screen.INVOICE_LIST)
    Box(Modifier.testTag(AppMapId.Element.INVOICE_ADD_BUTTON))      // ok: generated constant
    Box(Modifier.testTag("invoice.save.button"))                     // string_literal_id
  }
}

// AppMapId.Screen.LOGIN is never referenced here: marker_unreferenced for login on android when
// this file is the only Android source scanned.

File: tools/app-map-mcp/fixtures/lint/Bad.swift
// fixtures/lint/Bad.swift — lint-ids fixture (01 R8). Every violation is deliberate.
import SwiftUI

struct InvoiceListView: View {
  var body: some View {
    NavigationStack {
      List { Text("Invoices") }
        .accessibilityIdentifier("invoice.list.table")            // string_literal_id (must be AppMapID.Element.invoiceListTable)
      Button("New Invoice") {}
        .accessibilityIdentifier(AppMapID.Element.invoiceAddButton) // ok: generated constant
    }
    .accessibilityElement(children: .contain)
    .accessibilityIdentifier("screen.invoice_list")                 // string_literal_id (must be AppMapID.Screen.invoiceList)
  }
}

// Note: no view references AppMapID.Screen.clientPicker or AppMapID.Screen.invoiceDetail,
// so lint reports marker_unreferenced for client_picker and invoice_detail on ios when this
// file is the only iOS source scanned.
